tests/test-subscribers.js

```diff
@@ -1,10 +1,71 @@
 "use strict";
 
 const test = require("tape");
+const pg = require("pg");
 
-const gEmails = require("../subscribers");
+const AppConstants = require("../app-constants");
+const Subscribers = require("../subscribers");
+const DBUtils = require("../db-utils");
 
-test("gEmails is an object", (t) => {
-  t.plan(1);
-  t.equal(typeof(gEmails), "object");
+// eslint-disable-next-line no-process-env
+if (!process.env.TESTING_ENVIRONMENT) {
+  console.log("Attempting to run database tests without TESTING_ENVIRONMENT set, exiting.");
```

Comment:
Why the explicit env var check here?
Does this mean `npm run test` will fail unless we add `TESTING_ENVIRONMENT=1 npm test`?